packages/ui-library/src/components/form-label/index.stories.ts

```diff
@@ -56,13 +56,14 @@ export default {
     },
 
     hasError: {
+      name: 'hasError',
```

Comment:
Is this name required? I saw in some PR name property were removed for number of fields. Can you double check it?